src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -142,7 +141,7 @@ const RenderListFeed = (props) => {
   const topicBottomPosition = () => {
     if (hasComment) {
       if (getTotalReaction(item) === 1) {
-        return getHeightReaction() - 20;
+        return isBlurred && getHeightReaction() + normalize(4);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation of `topicBottomPosition` has been changed. The new code adds a normalized value of 4 to the reaction height when the post is blurred. This seems to be a design change, but please ensure that this change doesn't affect other parts of the layout.
```diff
- return isBlurred && getHeightReaction() + normalize(4);
+ return getHeightReaction() - 20;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->